src/AutocompleteBase.js

```diff
@@ -93,6 +93,16 @@ class AutocompleteBase extends Component {
  */
 AutocompleteBase.STATE = {
 	/**
+	 * A unique identification for the component. It is also used to compouse
+	 * list items' id.
+	 * @type {function}
+	 * @default Identity function.
```

Comment:
This is another comment I'd made, but didn't show up correctly in the pull request:

Identify functions are functions that return the same value they receive, which is not the case for `listId`. Also, it's not of type **function**, it actually receives a **string**. Can you fix these issues? It's also important to change the description for `listId` since it's not the component's identification, just the id used by the list's main element. Also, it should be **compose**, not **compouse** :)